inttest/common/footloosesuite.go

```diff
@@ -227,7 +210,7 @@ func (s *FootlooseSuite) SetupSuite() {
 		os.Exit(1)
 	}()
 
-	s.waitForSSH()
+	s.waitForSSH(context.Background())
```

Comment:
What about this, to match the previous behavior?

```suggestion
	s.waitForSSH(ctx)
```